.github/workflows/tests.yml

```diff
@@ -154,58 +154,59 @@ jobs:
           name: test-matrix
           path: semgrep/test-matrix.json
 
-  perf-test:
-    name: semgrep perf tests
-    runs-on: ubuntu-latest
-    needs: [build-core]
-    strategy:
-      matrix:
-        python: [3.7]
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          submodules: true
-      - name: Setup Python
-        uses: actions/setup-python@v1
-        with:
-          python-version: ${{ matrix.python }}
-      - name: Run 0.29.0 Timing Benchmark
-        run: |
-          pip3 install semgrep==0.29.0
-          semgrep --version
-          semgrep-core -version
-          python3 -m semgrep --version
-          export PATH=/github/home/.local/bin:$PATH
+  # perf-test:
```

Comment:
if you comment all of this, you need to put a comment why you did that.

Reply:
Oops I meant to delete this since you've added the other two tests to bench